editor/plugins/gdextension/gdextension_config_dialog.cpp

```diff
@@ -0,0 +1,237 @@
+	entry_symbol_edit->set_tooltip_text(TTR("Required. The symbol to use as the entry point for the extension."));
+	entry_symbol_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	grid->add_child(entry_symbol_edit);
+
+	// Compatibility Min Version
+	Label *compat_min_version_label = memnew(Label);
+	compat_min_version_label->set_text(TTR("Compat Min Version:"));
+	compat_min_version_label->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_RIGHT);
+	grid->add_child(compat_min_version_label);
+
+	compat_min_version_edit = memnew(LineEdit);
+	compat_min_version_edit->set_placeholder(VERSION_BRANCH);
+	compat_min_version_edit->set_tooltip_text(TTR("Required. The minimum version of Godot that the extension is compatible with."));
+	compat_min_version_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	grid->add_child(compat_min_version_edit);
+
+	// Compatibility Max Version
+	Label *compat_max_version_label = memnew(Label);
+	compat_max_version_label->set_text(TTR("Compat Max Version:"));
+	compat_max_version_label->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_RIGHT);
+	grid->add_child(compat_max_version_label);
+
+	compat_max_version_edit = memnew(LineEdit);
+	compat_max_version_edit->set_placeholder("<no maximum>");
+	compat_max_version_edit->set_tooltip_text(TTR("Optional. The maximum version of Godot that the extension is compatible with."));
+	compat_max_version_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	grid->add_child(compat_max_version_edit);
+
+	// Reloadable checkbox.
+	Label *reloadable_label = memnew(Label);
+	reloadable_label->set_text(TTR("Reloadable:"));
+	reloadable_label->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_RIGHT);
+	grid->add_child(reloadable_label);
+
+	reloadable_checkbox = memnew(CheckBox);
+	reloadable_checkbox->set_tooltip_text(TTR("Optional. Whether the extension can be reloaded at runtime. Recommended to be true for most extensions."));
+	reloadable_checkbox->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	grid->add_child(reloadable_checkbox);
+
+	Control *spacing = memnew(Control);
+	spacing->set_custom_minimum_size(Size2(0, 4 * EDSCALE));
+	vbox->add_child(spacing);
+
+	validation_panel = memnew(EditorValidationPanel);
+	validation_panel->set_custom_minimum_size(Size2(500, 120) * EDSCALE);
+	validation_panel->set_v_size_flags(Control::SIZE_SHRINK_END);
+	validation_panel->add_line(MSG_ID_ENTRY_SYMBOL_NAME, TTR("Entry symbol is valid."));
+	validation_panel->add_line(MSG_ID_COMPAT_MIN_VERSION, TTR("Compatibility minimum version is valid."));
+	validation_panel->add_line(MSG_ID_COMPAT_MAX_VERSION, TTR("Compatibility maximum version is valid."));
+	validation_panel->set_update_callback(callable_mp(this, &GDExtensionConfigDialog::_on_required_text_changed));
+	validation_panel->set_accept_button(get_ok_button());
+	vbox->add_child(validation_panel);
+
+	entry_symbol_edit->connect("text_changed", callable_mp(validation_panel, &EditorValidationPanel::update).unbind(1));
```

Comment:
```suggestion
	entry_symbol_edit->connect(SceneStringName(text_changed), callable_mp(validation_panel, &EditorValidationPanel::update).unbind(1));
```
New constant, for all